sdk/clientcore/tools/annotation-processor-dependency-tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Core Tests shared library for Java
```

Comment:
Can CI handle 4 levels deep packages? We generally use the pattern `sdk/<group>/<library>`. This has `sdk/<group>/tools/<library>`.

Reply:
yeah, I already see some issues in CI around it. I will confirm with eng sys on this. But do we have more preference to keep testing module and annotation-processor module directly under clientcore instead of tools?